swupd/packs.go

```diff
@@ -212,6 +233,15 @@ func WritePack(w io.Writer, fromManifest, toManifest *Manifest, outputDir, chroo
 		fullChrootDir = filepath.Join(chrootDir, fmt.Sprint(toVersion), "full")
 	}
 
+	if fromManifest != nil {
+		dManifest = &Manifest{
+			Header: fromManifest.Header,
+			Name:   fromManifest.Name,
+		}
+		dManifest.Header.ContentSize = 0
+		dManifest.Header.Includes = nil
```

Comment:
Please add that comment I asked for describing why we are setting this to `nil` and point out the fact that if we ever need to add it back in it does *not* require a format bump for the client.